utils/context/keys.go

```diff
@@ -32,6 +32,8 @@ const (
 	RatiosCacheExpiryDurationCTXKey CTXKey = "ratios_client_cache_expiry"
 	// RatiosCachePurgeDurationCTXKey - context key for ratios client cache purge
 	RatiosCachePurgeDurationCTXKey CTXKey = "ratios_client_cache_purge"
+	// LogEvent holds a log event to be passed through settlement functions
+	LogEvent CTXKey = "log_event"
```

Comment:
this log event is a zerolog log event? is there a reason we are not passing a sub logger instead?

Reply:
fixed in 7f0ed18